extensions/wikia/Search/classes/Test/HooksTest.php

```diff
@@ -85,6 +89,11 @@ public function testOnArticleDeleteComplete() {
 		    ->with   ( 123 )
 		    ->will   ( $this->returnValue( true ) )
 		;
+		$mockIndexer
+			->expects( $this->once() )
+			->method ( 'canIndex' )
+			->will   ( $this->returnValue( true ) )
+		;
 		$this->mockClass( 'Wikia\Search\Indexer', $mockIndexer );
 		$this->assertTrue(
 				$mockHooks->onArticleDeleteComplete( $mockArticle, $mockUser, 'why not', 123 )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "why not" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)